### PR TITLE
Add ESLint section to test-metadata.md

### PR DESCRIPTION
**Pull request by Awk34**

### CODE REVIEW

File: jekyll/_cci1/test-metadata.md [hunk 1]
```diff
@@ -156,6 +157,21 @@ test:
 
 
 
+#### <a name="eslint"></a>ESLint
+
+To output JUnit results from [ESLint](http://eslint.org/), you can use the [JUnit formatter](http://eslint.org/docs/user-guide/formatters/#junit).
+
+A working [circle.yml]( {{ site.baseurl }}/1.0/configuration/) section for testing might look like this:
```
Comment (smart-alek) [inline jekyll/_cci1/test-metadata.md]: Edit:

A working [circle.yml]( {{ site.baseurl }}/1.0/configuration/) `test` section might looke like this:

File: jekyll/_cci1/test-metadata.md [hunk 2]
```diff
@@ -156,6 +157,21 @@ test:
...
+
+```
+test:
+  pre:
+    - mkdir -p $CIRCLE_TEST_REPORTS/reports
+    - eslint ./src/ --format junit --output-file $CIRCLE_TEST_REPORTS/reports/eslint.xml
+```
+
+
+
```
Comment (smart-alek) [inline jekyll/_cci1/test-metadata.md]: Do you need all this whitespace, or is this standard throughout the doc?

Comment (Awk34) [inline jekyll/_cci1/test-metadata.md]: I just copied the AVA section above with all it's whitespace, it probably doesn't need it

**Comment by Awk34** [comment]
Side note: I now have the circleci-docs builds showing up on my CircleCI dashboard, and there doesn't seem to be an 'Unfollow Project' button anywhere. Is there any way to prevent them from cluttering up my dashboard?

**Comment by smart-alek** [comment]
@Awk34 If you click on the "circleci" org, that should give you a list of associated projects. Find `circleci-docs` in there and click the "Unfollow project" to the right of it.

Let me know if you have any other questions, and thanks again for the contribution!

**Pull request merged**